object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,11 @@
+/* exported toObject */
+// Define a function named toObject that takes an array of two elements as an argument.
+// The function should return an object with the first element as the key and the second element as the value.
+//    define an empty object to a variable named Object
+//    assign the second element of the array to Object[firstElement]
+//    return the Object
+function toObject(array) {
+  var Object = {};
```

Comment:
Make sure to avoid making variable with Capital letters, there is already a `Object` class we don't want to overwrite!